onboarding-carts/shipyard.yaml

```diff
@@ -80,3 +80,17 @@ spec:
               properties:
                 deploymentstrategy: "direct"
             - name: "release"
+
+        - name: "remediation"
+          triggeredOn:
+            - event: "production.remediation.finished"
+              selector:
+                match:
+                  evaluation.result: "fail"
+          tasks:
+            - name: "get-action"
+            - name: "action"
+            - name: "evaluation"
+              triggeredAfter: "10m"

```

Comment:
please also update the spec version of the shipyard file

Reply:
Good catch, will do